src/nodejs-common/service.ts

```diff
@@ -13,33 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import {
```

Comment:
Is this file even necessary anymore?

Reply:
Probably not. I was going to sweep up all the garbage once I got things back to a working state.

Reply:
sg, can you add it to some tracking list of TODOs